Teaching/Blockchain/crypto.md

```diff
@@ -0,0 +1,39 @@
+# Exploring Public-Private Key Encryption and Bitcoin Wallets
+
+## Background
+
+Public-private key encryption is a type of cryptography that provides a secure way to transmit and store data. It is based on the mathematical concept of asymmetric encryption, which uses a pair of keys to encrypt and decrypt data. The keys are related mathematically, but it is computationally infeasible to deduce one key from the other. This means that the encryption is highly secure and virtually unbreakable.
+
+Public-private key encryption is widely used in modern online communication to ensure the confidentiality and integrity of data. In the context of cryptocurrencies such as Bitcoin, public-private key encryption is used to secure and manage Bitcoin wallets.
+
+A Bitcoin wallet is a digital wallet that stores Bitcoin and allows users to send and receive Bitcoin transactions. Each Bitcoin wallet is associated with a pair of public and private keys, which are used to sign and verify transactions. Public keys are used to receive Bitcoin, while private keys are used to spend Bitcoin.
+
+Overall, public-private key encryption is an important tool for securing online communication and transactions. It provides a high level of security and has a wide range of applications in modern technology.
+
+## Tasks
+
+Research and gather information on the key concepts related to Public-Private Key Encryption and Bitcoin Wallets
+
+Create a presentation with the following content:
+- The basics of public-key cryptography and how it differs from traditional symmetric-key cryptography.
+- How public and private keys are generated and used to encrypt and decrypt data.
+- The concept of digital signatures and how they are used to verify the authenticity of data.
+- How public-private key encryption is used in Bitcoin wallets to secure and manage Bitcoin transactions.
+- The different types of Bitcoin wallets, including hot wallets and cold wallets, and the advantages and disadvantages of each.
+- Best practices for securing Bitcoin wallets, including the importance of backing up private keys and using multi-factor authentication.
```

Comment:
Add another point:

- [Optional, but recommended] Use the `bitcoin-cli` client to make a regtest wallet, and try creating transactions.

Reply:
Falls under hands-on demonstration.